InvenTree/order/models.py

```diff
@@ -1086,7 +1089,7 @@ def issue_order(self):
         )
 
     @transaction.atomic
-    def complete_order(self, user, **kwargs):
+    def ship_order(self, user, **kwargs):
```

Comment:
Not sure if this is not breaking backwards combat for plugins - thoughts @SchrodingersGat @wolflu05?

Reply:
It could - but any plugin which calls this method will need to know about the new status code too. I'm not sure if this is something we should account for (e.g. with a deprecation warning) or just leave

Reply:
At least a warning in the release notes and a breaking label is waranted

Reply:
I think that should be sufficient.